stostools/NuGet.Config

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<add key="NugetOfficialV3" value="https://api.nuget.org/v3/index.json" />
+		<add key="Local" value="tools/LocalNugetFeed" />
+		<add key="deshankfeed" value="\\scratch2\scratch\deshank\nugetfeed" />
```

Comment:
Are you sure you want a reference to \\scratch2\ here?

Reply:
Yes that is the best option we have right now for a quick turnaround time.
But the plan is to have a nuget package in nuget.org signed a published.
But this early in this cycle is not an optimum solution when we are doing this for the first time.

But rest assured, the only way we want to roll out is in the form of a signed nuget package (versioned)